### PR TITLE
Separating rsyslog_viaq_k8s_packages from rsyslog_viaq_packages.

### PR DESCRIPTION
**Pull request by nhosoi**
Moving module(load="mmjsonparse") from 10-viaq_main.conf to 10-mmk8s.conf.

Solving https://github.com/linux-system-roles/logging/issues/23.

### CODE REVIEW

**Comment by sradco** [comment]
/lgtm
I see users can use "rsyslog_packages" parameter if additional packages are required.

File: roles/rsyslog/README.md [hunk 1]
```diff
@@ -185,8 +185,47 @@ rsyslog_user: root
...
 
+To receive remote input, you could add 'network' to rsyslog_capabilities to the basic variables as follows.
+```
+rsyslog_capabilities: [ 'network' ]
+```
+It will configure imudp as well as imptcp.
+
+To put logs from the remote input in the separate files, e.g., /var/log/remote/hosts/%HOSTNAME%/*.log, 'remote-files' could be added to rsyslog_capabilities.
+```
+rsyslog_capabilities: [ 'network', 'network' ]
```
Comment (sradco) [inline roles/rsyslog/README.md]: You have a bug here ...
Should be 
rsyslog_capabilities: [ 'network', 'remote-files' ]

@nhosoi

File: roles/rsyslog/README.md [hunk 2]
```diff
@@ -185,8 +185,47 @@ rsyslog_user: root
...
+
+Finally, to make the network communication safe, enable rsyslog_pki and add 'tls' to rsyslog_capabilities.
+```
+rsyslog_enabled: true
+# install example packages & config files
+rsyslog_example: true
+rsyslog_pki: true
+rsyslog_group: root
+rsyslog_user: root
+rsyslog_capabilities: [ 'network', 'network', 'tls' ]
```
[the remark this answers is not in the document]
Comment (sradco) [reply roles/rsyslog/README.md]: See the line with "TASK [rsyslog : create rsyslog viaq subdir]" in the example section. It should be removed.
Also, there is a spelling error in "TASK [rsyslog : Generate exaple configuration files in rsyslog.d]" for the word "example.
@nhosoi

**Comment by nhosoi** [comment]
Thanks for the reviews, @sradco.  I fixed them, squashed and pushed the new patch.

**Pull request merged**